cmd/utils/flags.go

```diff
@@ -1594,6 +1594,17 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 	setTxResendConfig(ctx, cfg)
 }
 
+func MakeGenesis(ctx *cli.Context) *blockchain.Genesis {
+	var genesis blockchain.Genesis
```

Comment:
@hyochan-brown Can I ask why you didn't declare `genesis` as a pointer?

Reply:
It's my mistake...
Thank you so much.
Not using pointer will skip the nil check and nothing came out in command.